src/sets.jl

```diff
@@ -102,6 +102,9 @@ struct Interval{T <: Real} <: AbstractScalarSet
     lower::T
     upper::T
 end
+Interval(s::GreaterThan) = Interval(s.lower, typemax(s.lower))
```

Comment:
What do we do for `GreaterThan{Bool}` or `GreaterThan{Int}` ? In MathProgBase, we [throw a warning in that case](https://github.com/JuliaOpt/MathProgBase.jl/blob/7c6f0a8414fdc018cf7ba90eb3c5f29c2e68a738/src/HighLevelInterface/HighLevelInterface.jl#L6-L10).
We could also display a warning for non-`AbstractFloat` type or simply restrict this definition to `AbstractFloat` (I would prefer this second approach).

Reply:
Good point. I also prefer the second approach.